### PR TITLE
New helper functions added, new tests added

### PR DESCRIPTION
**Pull request by CerealKiller97**
Signed-off-by: Stefan Bogdanovic <sbogdanovic@nanointeractive.com>

### CODE REVIEW

File: testing/http/modifier.go [hunk 1]
```diff
@@ -28,9 +28,32 @@ func WithHeaders(t testing.TB, headers http.Header) RequestModifier {
...
 }
 
+// WithQuery Provides a way to put query string params.
+//
+//	WithQuery(map[string]string{
+//		"id": "mongoid",
+//	})
+//
+// Returns URL?id=mongoid
+func WithQuery[T any](t *testing.T, queryMap map[string]string) RequestModifier {
```
Comment (CodeLieutenant) [inline testing/http/modifier.go]: change to testing.TB

File: testing/http/modifier.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 
 type RequestModifier func(*http.Request) *http.Request
 
-func WithHeaders(t testing.TB, headers http.Header) RequestModifier {
```
Comment (CodeLieutenant) [inline testing/http/modifier.go]: Return testing.TB

File: testing/http/modifier.go [hunk 3]
```diff
@@ -42,7 +65,7 @@ func WithBody[T any](t testing.TB, body T) RequestModifier {
 	}
 }
 
-func WithCookies(t testing.TB, cookies []*http.Cookie) RequestModifier {
```
Comment (CodeLieutenant) [inline testing/http/modifier.go]: Return testing.TB

File: testing/testing.go [hunk 4]
```diff
@@ -8,7 +8,11 @@ import (
 
 	"github.com/spf13/viper"
 
-	utils "github.com/nano-interactive/go-utils"
+	"github.com/nano-interactive/go-utils"
+)
+
+var (
+	ErrConfigNotFound = errors.New("config file not found")
```
Comment (CodeLieutenant) [inline testing/testing.go]: remove this line and inline it as this code is only for testing

**Review by CodeLieutenant — CHANGES_REQUESTED** [review]

File: testing/http/modifier.go (hunk 2 above)
Comment (CerealKiller97) [reply testing/http/modifier.go]: Resolved.

File: testing/http/modifier.go (hunk 1 above)
Comment (CerealKiller97) [reply testing/http/modifier.go]: Resolved.

File: testing/http/modifier.go (hunk 3 above)
Comment (CerealKiller97) [reply testing/http/modifier.go]: Resolved.

File: testing/testing.go (hunk 4 above)
Comment (CerealKiller97) [reply testing/testing.go]: Resolved.

**Review by CodeLieutenant — APPROVED** [review]

**Pull request merged**